fileglancer/handlers.py

```diff
@@ -39,6 +39,29 @@ def get_current_user(self):
         """
         return os.getenv("USER", self.current_user.username)
 
+    def get_home_directory_path(self):
+        """
+        Get the home directory path of the current user.
+        
+        Returns:
+            str: The home directory path.
+        """
+        return os.path.expanduser(f"~{self.get_current_user()}")
+
+    def get_home_file_share_path_name(self):
+        """
+        Get the file share path for the current user's home directory.
+        
+        Returns:
+            str: The file share path name for the user's home directory, formatted as file_share_path_name
+        """
+        home_file_share_path = os.path.split(self.get_home_directory_path())[0]
+        file_share_paths = get_fsp_manager(self.settings).get_file_share_paths()
+        for fsp in file_share_paths:
```

Comment:
Was there a discussion of caching the result from this loop, so that it only had to be performed once? Is the caching done somewhere else in the code?